assessments/projects/coredns/self-assessment.md

```diff
@@ -0,0 +1,267 @@
+# Self-assessment
+Authors: Tom Zhang, Rohit Chaudhari, Maryam Mohagheghi, Vamshi Madineni
+
+Contributors/Reviewers: Pranava Kumar Vemula (@Rana-KV).
+
+This document evaluates the design goals for [CoreDNS](https://coredns.io/) as provides a security assessment of the software.
+
+# Self-assessment outline
+
+## Table of contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+* [Overview](#overview)
+  * [Actors](#actors)
+  * [Actions](#actions)
+  * [Background](#background)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Self-assessment use](#self-assessment-use)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+* [Appendix](#appendix)
+
+## Metadata
+
+A table at the top for quick reference information, later used for indexing.
+
+|   |  |
+| -- | -- |
+| Software | [CoreDNS](https://github.com/coredns/coredns) |
+| Security Provider | No |
+| Languages | Golang |
+| | |
+
+### Security links
+
+| Doc | url |
+| -- | -- |
+| Core53 | [Audit Report](https://coredns.io/assets/DNS-01-report.pdf) |
+| Trail of Bits | [Audit Report](https://github.com/trailofbits/publications/blob/master/reviews/CoreDNS.pdf) |
```

Comment:
Please include documents related to security procedures and default configurations of this project. This includes SECURITY.md among others - https://github.com/coredns/coredns/security